src/imc_handle.cpp

```diff
@@ -27,7 +27,7 @@ void try_callback(const IMC::Message* imc_msg)
 IMCHandle::IMCHandle(const std::string& tcp_addr, const std::string& tcp_port,
                      const std::string& sys_name, int imc_id)
     : udp_link(std::bind(&IMCHandle::tcp_callback, this, std::placeholders::_1),
-               tcp_addr, tcp_port),
+               tcp_addr, tcp_port, imc_id),
```

Comment:
I don't see this used anywhere (imc_id)?

Reply:
It is used in UDPLink::publish_multicast and UDPLink::publish.

Reply:
Ah, OK, I see now.